test/express-app/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "express-app",
   "description": "express-app",
-  "version": "0.0.0"
+  "version": "0.0.0",
+  "main": "server.js"
```

Comment:
Did you find this necessary? server.js is one of the defaults, it shouldn't have to be here. I can slc run this app without a main key.